a-braham/Questioner
# #163331516] Changed home file name, changed html

#### What does this PR do?
 * The pull request creates index.html file from home.html

#### Description of Task to be completed?
  **Have the following features complete**
  * Landing page should be simple and descriptive
  * Page should be responsive
  * Should not have any inline css

#### How should this be manually tested?
  * Clone the repository
  * Open the index.html file with chrome
  * Test for responsiveness, check the code for error
 
#### Screenshots
![screenshot from 2019-01-19 01-28-16](https://user-images.githubusercontent.com/45788436/51416212-8e194100-1b89-11e9-9881-b6534f490778.png)

#### What are the relevant pivotal tracker stories?
  #163331516
